### PR TITLE
Fix typos and capitalization style

### PR DESCRIPTION
**Pull request by ashawley**
- Fix a few spelling typos
- Use capitalization for first-level headings consistent
- Don't use capitalization for second-level headings

### CODE REVIEW

File: editors/emacs/userguide.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ The extra sbt integration enables some convenient workflow options:
 
 ## Editing
 
-### Error Highlighting
+### Error highlighting
```
Comment (fommil) [inline editors/emacs/userguide.md]: lowercase here but upper elsewhere?

Comment (fommil) [inline editors/emacs/userguide.md]: ah, third level

**Comment by fommil** [comment]
thanks!

**Comment by fommil** [comment]
I seen that github recently changed how they render links... if you'd like to improve the site a little further we could do with fixes to all our image links :crying_cat_face:

File: editors/emacs/userguide.md (hunk 1 above)
Comment (ashawley) [inline editors/emacs/userguide.md]: Yeah, sorry, for introducing the off-by-one error from my brain to yours

**Comment by ashawley** [comment]
> if you'd like to improve the site a little further we could do with fixes to all our image links

Ah, I see what you mean...

http://ensime.org/build_tools/

**Pull request merged**